Makefile

```diff
@@ -113,22 +120,13 @@ copyapi: api
 
 # Build resource reference
 genresources:
-	make -C gen-resourcesdocs kwebsite 
-
+	make -C gen-resourcesdocs kwebsite
 
 # Build config API reference
 CONFIGSRC=genref/output/md
 CONFIGDST=$(WEBROOT)/content/en/docs/reference/config-api/
 configapi:
 	make -C genref
 
-copy-configapi: config-api
-	cp $(CONFIGSRC)/apiserver-audit.v1.md $(CONFGDST)
-	cp $(CONFIGSRC)/apiserver-webhookadmission.v1.md $(CONFGDST)
-	cp $(CONFIGSRC)/client-authentication.v1beta1.md $(CONFGDST)
-	cp $(CONFIGSRC)/kube-proxy-config.v1alpha1.md $(CONFGDST)
-	cp $(CONFIGSRC)/kube-scheduler-config.v1beta1.md $(CONFGDST)
-	cp $(CONFIGSRC)/kube-scheduler-policy-config.v1.md $(CONFGDST)
-	cp $(CONFIGSRC)/kubelet-config.v1beta1.md $(CONFGDST)
-	cp $(CONFIGSRC)/kuberc.v1alpha1.md $(CONFGDST)
-	cp $(CONFIGSRC)/kuberc.v1beta1.md $(CONFGDST)
+copyconfigapi: configapi
+	cp $(CONFIGSRC)/*.md $(CONFIGDST)/
```

Comment:
The collection of so called "config APIs" is larger than what we publish on k8s/website.
The reason is that not all markdowns are referenced from the website pages.
It is fine to copy them all over, but a tighter control might be better.
We can leave that to a future PR.

Reply:
My goal was to ensure that new ones are copied over, and when creating a PR against k/website that's where you can decide what should and should not be included.